RELEASE_NOTES.md

```diff
@@ -1,5 +1,23 @@
 # Release Notes
 
+## 2.1.19 "TBD" - TBD
+
+* This release adds support for capturing Kubernetes container names to the kubernetes monitor via log line attributes.
```

Comment:
"This release add support for adding the Kubernetes container name as an attribute to all log lines ingested via the Kubernetes monitor.